### ISSUE TITLE
custom hover tooltips should be shown on keyboard focus

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#35079]
Verify that for icon labels with custom hovers, for example, the editor action ones, the tooltip shows up on keyboard focus of the item. 
![Screenshot 2023-11-27 at 2 36 14 PM](https://github-com/microsoft/vscode/assets/29464607/9320d2a9-c0eb-4393-8cea-1776d8cf2e62)

_Originally posted by -meganrogge in https://github-com/microsoft/vscode/-/132344-issuecomment-1828560148_
